### ISSUE TITLE
Manually Identify Episode with correct .tbn

### ISSUE THREAD

**Issue by michou83** [boxeehacks/boxeehack#167]
Currently when manually identifying a TV episode, the NFO data is displayed correctly for the identified episode, but the thumbnail will display the TV show's folder.jpg image instead of the corresponding TBN file image. 

So when manually adding a TV episode the corresponding TBN should get loaded as the thumbnail.

**Comment by war59312** [comment]
Not sure if its possible to fix this one atm. I had asked about this a few weeks ago.

**Comment by boxeehacks** [comment]
Yeah this is probably not possible. But will look into it nevertheless...

**Comment by gadgetchnnel** [comment]
I've been experimenting with a shell script which queries the database to find all TV show episodes with missing thumbnails, checks for a matching TBN file for that episode, and then updates the database to set the thumbnail location to the location of the TBN file. The script takes a while to process everything, but I think there are ways that this can be improved (such as only querying episodes which have been updated recently in the database). A script like this could be scheduled to run regularly.
Even if this could be done within one of the skin files, it's probably better to do it like this as the thumbnail location would be stored in the database rather than needing to be dynamically found every time to view the episodes for the show.
